boilerplate/src/views/shared/text-field/text-field.story.tsx

```diff
@@ -89,3 +89,32 @@ storiesOf("TextField", module)
       </UseCase>
     </Story>
   ))
+  .add("Ref Forwarding", () => (
+    <Story>
+      <UseCase
+        text="Ref Forwarding"
+        usage=""
+      >
+        <State initial={{ value: "fancy colour" }}>
+          {({ state, setState }) => (
+            <TextField
+              onChangeText={value => setState({ value })}
+              value={state.value}
+              label="Name"
+              inputStyle={{
+                backgroundColor: "rebeccapurple",
+                color: "white",
+                padding: 40,
+                borderWidth: 10,
+                borderRadius: 4,
+                borderColor: "hotpink",
+              }}
+              forwardedRef={ref => ref && ref.focus && ref.focus()}
+              onFocus={() => console.tron.log("focus")}
```

Comment:
I'd recommend changing onFocus in this test from "console.tron.log to window.alert"

Reply:
@danielberkompas I added [this logic](https://github.com/infinitered/ignite-ir-boilerplate-bowser/pull/107/files#diff-54c8967cab722409eb6ee06657294ec7R151) because closing the alert causes the text field to become focused, which triggers the alert which...